README.md

```diff
@@ -0,0 +1,8 @@
+# MOA Command Line Tool
+
+This project contains the `moactl` command line tool that simplifies the use of
+the _MOA_.
+
+## Quickstart guide
+
+Get started with moa by following the [Quickstart guide](moa-quickstart.md).
```

Comment:
I think this file should contain the installation part, rather (or in addition) to the quickstart.md

Reply:
I'll move it. Sounds good to me.